plugins/mcreator-core/procedures/entity_set_saturation.json

```diff
@@ -0,0 +1,29 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    },
+    {
+      "type": "input_value",
+      "name": "amount",
+      "check": "Number"
+    }
+  ],
+  "inputsInline": true,
+  "previousStatement": null,
+  "nextStatement": null,
+  "colour": 195,
```

Comment:
195 is color for entity procedures, not for player procedures. Check the colors of your blocks and make sure they follow conventions.

Reply:
done!